src/entities/activity/model/mappers.input.mock.ts

```diff
@@ -10,6 +10,7 @@ const basicAppletDetails = {
   isSkippable: false,
   isReviewable: false,
   isHidden: false,
+  autoAssign: false,
```

Comment:
When assignments feature flag is enabled in production, all existing activities and flows will have `autoAssign: true` by default. Also, new activities and flows will have `autoAssign: true`. In the web apps where I've had to add the `autoAssign` property to mock activity data, I've defaulted that property value to `true` because it's the default value as well as the more common use case to test (until people actually start using manual assignments). Wondering what your reason is for choosing `false` here and in all other mocks by default. Perhaps it has no actual impact in the end since the existing unit tests aren't doing anything with that value.

Reply:
I was mostly trying to prevent too many tests from failing. But that was just a preemptive measure. I feel it should be okay to change this to `true`. I'll give it a try.